packages/camera/camera_android_camerax/example/lib/main.dart

```diff
@@ -54,20 +54,23 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
   VideoPlayerController? videoController;
   VoidCallback? videoPlayerListener;
   bool enableAudio = true;
-  double _minAvailableExposureOffset = 0.0;
-  double _maxAvailableExposureOffset = 0.0;
   double _currentExposureOffset = 0.0;
   late AnimationController _flashModeControlRowAnimationController;
   late Animation<double> _flashModeControlRowAnimation;
   late AnimationController _exposureModeControlRowAnimationController;
   late Animation<double> _exposureModeControlRowAnimation;
   late AnimationController _focusModeControlRowAnimationController;
   late Animation<double> _focusModeControlRowAnimation;
-  double _minAvailableZoom = 1.0;
-  double _maxAvailableZoom = 1.0;
+
   double _currentScale = 1.0;
   double _baseScale = 1.0;
 
+  // TODO(camsim99): Make these configuration values variable.
```

Comment:
If you have a bug for this already written consider including it.

Reply:
Did you intend to add bug here? If so I do not see one.

Reply:
I will revert this change, as I am implementing this behavior in https://github.com/flutter/packages/pull/3798